en/engine/animation/skeletal-animation.md

```diff
@@ -0,0 +1,73 @@
+# Skeletal animation
+
+__Skeletal Animation__ is a common but special type of animation. Two different systems are provided and each is optimized for different purposes. Seamless switching between these two systems can be as simple as toggle the `useBakedAnimation` switch on `SkeletalAnimationComponent`, even at runtime. when enabled, the pre-baked system will be used, or the real-time calculated system if otherwise.
+
+## Pre-baked Skeletal Animation System
+
+The dominant purpose of this system is performance, and some sacrifices in expressiveness are also considered acceptable. __Cocos Creator 3D__ has made many low-level optimizations in a targeted manner. The current runtime process is roughly as follows:
+  * All animation data will be pre-sampled in advance according to the specified frame rate and baked onto a global-managed joint texture atlas.
+  * Depending on whether the operating platform supports floating-point textures, the corresponding texture format will be RGBA32F, or automatically fallback to RGBA8 if not available (The rendering results should be identical, it is only the fail-safe approach for really low-end devices, and shouldn't be of any concern for game developers).
+  * Each __Skeletal Animation Component__ (`SkeletalAnimationComponent`) is responsible for maintaining the current playback progress, stored in the form of UBO (a `vec4`).
+  * Each skinning model component (`SkinningModelComponent`) holds a pre-baked skinning model class (`BakedSkinningModel`). Based on the bounding box information pre-baked in the same way to do frustum culling, update the UBO, and get the current data from the texture atlas on the GPU to complete the skinning.
+
+## Real-time calculated Skeletal Animation System
+
+The dominant purpose of this system is expressiveness, ensuring the correct display of all details, and complete program control capabilities.
+
+The current runtime process is roughly as follows:
+  * All animation data are calculated dynamically according to the current global time;
+  * Animation data will be output to the skeleton node tree of the scene;
+  * Users and any other system can affect the skin effect by manipulating this bone node tree;
```

Comment:
> We follow the convention of using `joint` as the standard word for `骨骼` in code, so better to keep it in the docs

> this bone node tree

By that I mean all references to `bone` should be replaced with `joint`, although this particular case here I think it'd better if just omitting it

Reply:
replaced all instances of __bone__ with __joint__

removed __joint__ from this sentence